### PR TITLE
Hare ears

### PR DESCRIPTION
**Pull request by miguelop1**
Hare ears

# About The Pull Request
We add Hare ears. Created by o0xroxasx0o2600

## Why It's Good For The Game
More ears moment

## A Port?
no

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added more ears
:cl:

### CODE REVIEW

**Comment by miguelop1** [comment]
Aint working yet.

**Comment by Kilmented** [comment]
> ain't working yet
famous last words

**Comment by Tk420634** [comment]
> Aint working yet.

caught in 4k

File: code/modules/mob/dead/new_player/sprite_accessories/ears.dm [hunk 1]
```diff
@@ -456,6 +456,20 @@
 	icon_state = "bunnytall"
 	matrixed_sections = MATRIX_RED_GREEN
 
+/datum/sprite_accessory/ears/human/bunnylarge
+    name = "Hare (Tall)"
+    icon = 'modular_citadel/icons/mob/32x64_mam_ears.dmi'
+    icon_state = "hare"
+	color_src = MATRIXED
```
Comment (BongaTheProto) [inline code/modules/mob/dead/new_player/sprite_accessories/ears.dm:463]:
```suggestion
    icon_state = "hare"
    color_src = MATRIXED
```

File: code/modules/mob/dead/new_player/sprite_accessories/ears.dm [hunk 2]
```diff
@@ -456,6 +456,20 @@
...
+    matrixed_sections = MATRIX_RED_GREEN
+
+/datum/sprite_accessory/ears/mam_ears/bunnylarge
+    name = "Hare (Tall)"
+    icon = 'modular_citadel/icons/mob/32x64_mam_ears.dmi'
+    icon_state = "hare"
+	color_src = MATRIXED
```
Comment (BongaTheProto) [inline code/modules/mob/dead/new_player/sprite_accessories/ears.dm:470]:
```suggestion
    icon_state = "hare"
    color_src = MATRIXED
```

**Review by BongaTheProto — CHANGES_REQUESTED** [review]

File: code/modules/mob/dead/new_player/sprite_accessories/ears.dm (hunk 2 above)
Comment (miguelop1) [reply code/modules/mob/dead/new_player/sprite_accessories/ears.dm]: Alright the stupid shit fixed

**Comment by BongaTheProto** [comment]
Please modularize your code into modular_splurt

**Pull request merged**